src/platform-includes/profiling/index/why-profiling/javascript.mdx

```diff
@@ -0,0 +1,5 @@
+## Why profiling?
+
+With [profiling](/product/profiling/), Sentry tracks your software's performance by sampling your program's call stack in a variety of environments, collecting function level information about your code and enables you to fine tune the performance of your program. Sentry profiling captures function calls and their exact locations, aggregates them and shows you the most common code paths of your program, which surfaces areas of optimization opportunity that can help you increase both the performance of your code and increase user satisfaction.
+
+In a browser environment, profiling can help you pinpoint causes of UI jank and surface why values like input to next paint are performing poorly or why a long task was blocking the browser from repainting the screen and causing frame drops. All of this information enables you to fix real world performance issues and deliver a snappier user experience to your users.
```

Comment:
```suggestion
In a browser environment, profiling can help you pinpoint causes of UI jank, surface why values like interaction to next paint (INP) are performing poorly, or why a long task was blocking the browser from repainting the screen and causing frame drops. All of this information enables you to fix real world performance issues and deliver a snappier user experience to your users.
```

It's a bit of a runon sentence :D but this is so much better, it's much more clear why i would want this. I was really just confused is the input to next paint something different than INP, which is interaction to next pain?